<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChronoEdit</class>
 <widget class="QWidget" name="ChronoEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChronoEdit - Éditeur de chronologie de vol</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>&amp;Chronologie</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Ajouter un évenement</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QFormLayout" name="formLayout">
              <property name="fieldGrowthPolicy">
               <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="titreLabel">
                <property name="text">
                 <string>Titre</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="titreLineEdit"/>
              </item>
              <item row="1" column="1">
               <widget class="QSpinBox" name="heureSpinBox">
                <property name="minimum">
                 <number>-14400</number>
                </property>
                <property name="maximum">
                 <number>14400</number>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="lieuLabel">
                <property name="text">
                 <string>Lieu</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QComboBox" name="lieuCombo"/>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="actionnaireLabel">
                <property name="text">
                 <string>Actionnaire</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QComboBox" name="actionnaireCombo"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="heureLabel">
                <property name="text">
                 <string>Temps</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="plainTextEdit"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QPushButton" name="pushButton">
                <property name="text">
                 <string>&amp;Ajouter</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>&amp;Contributeurs</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="setContributeur"/>
         </item>
         <item>
          <widget class="QPushButton" name="addContributeur">
           <property name="text">
            <string>&amp;Ajouter contributeur</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="Contributeurs"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>&amp;Lieux</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="setLieu"/>
         </item>
         <item>
          <widget class="QPushButton" name="addLieu">
           <property name="text">
            <string>&amp;Ajouter lieu</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="Lieux"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>setContributeur</sender>
   <signal>returnPressed()</signal>
   <receiver>addContributeur</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>setLieu</sender>
   <signal>returnPressed()</signal>
   <receiver>addLieu</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>498</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
